components/network/resourcepack_info.yaml

```diff
@@ -0,0 +1,11 @@
+ResourcePackInfo:
+  type: object
+  properties:
+    url:
+      type: string
+      description: The URL to the resource pack
+      example: https://object-storage.svc.atl1.cox.minehut.com/v1/presigner/getpack/2024-12-11-14h-35m-19s_e7270247392410e2c2fc10abb7c6f7f0516acafe.zip
```

Comment:
Not sure how I feel with having a direct download link here, maybe just skip an example for this field

Reply:
Yeah thats fair. Just removed it.